server.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express')
+const app = express()
+const port = 3000
+const mysql = require('mysql2')
+
+// Import Sequelize Model
+const model = require('./models')
+const villains = require('./villains.js')
+const { Op } = require('sequelize')
+
+// App config
+app.set('view engine', 'pug')
+app.use(express.static('public'))
+app.use(express.json())
+app.use(express.urlencoded({
+  extended: true
+}));
+
+// Import Villains.js into SQL Database
+// model.villains.bulkCreate(villains)
+//   .then(() => {
+//     console.log('Succes')
+//   })
+
+
+// Get all villains
+app.get('/villains', async (req, res) => {
+  const villains = await model.villains.findAll()
+
+  res.send(villains)
+})
+
+// Get villain by slug
+app.get('/villains/:slug', async (req, res) => {
+  const { slug } = req.params
+
+  const villains = await model.villains.findAll({
```

Comment:
why findAll by slug insted of findOne. do you expect multiple items to have same slug?

also if nothing found, you return empty array. should you rather throw 404 not found